src/libraries/Common/src/Interop/Windows/Shell32/Interop.SHGetKnownFolderPath.cs

```diff
@@ -24,290 +24,390 @@ internal static class KnownFolders
             /// <summary>
             /// (CSIDL_ADMINTOOLS) Per user Administrative Tools
             /// "%APPDATA%\Microsoft\Windows\Start Menu\Programs\Administrative Tools"
+            /// {724EF170-A42D-4FEF-9F26-B60E846FBA4F}
             /// </summary>
-            internal const string AdminTools = "{724EF170-A42D-4FEF-9F26-B60E846FBA4F}";
+            internal static ReadOnlySpan<byte> AdminTools =>
+                [0x70, 0xF1, 0x4E, 0x72, 0x2D, 0xA4, 0xEF, 0x4F, 0x9F, 0x26, 0xB6, 0x0E, 0x84, 0x6F, 0xBA, 0x4F];
```

Comment:
We can keep the readable form with `"...."u8` strings for `ROS<byte>`.

https://github.com/dotnet/runtime/blob/fae90a6803c5a20b86d8183bf4bee702a0ee8239/src/libraries/System.Text.Json/src/System/Text/Json/JsonConstants.cs#L49

Reply:
I don't think so, a GUID is a 16-byte binary representation whose contents are not constrained to be printable characters.

Reply:
Ah, right. Its the internal representation.

Reply:
We can’t rely on "...“u8 strings for arbitrary byte values. For example, "\xF1" is interpreted as the Unicode character ñ, which UTF-8 encodes as two bytes: 0xC3 0xB1, not the single byte 0xF1. So the resulting ReadOnlySpan<byte> won’t match the intended byte sequence.